plugins/gs.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: gs
+spec:
+  version: v0.5.0
+  homepage: https://github.com/giantswarm/kubectl-gs
+  shortDescription: Giant Swarm plug-in for kubectl
```

Comment:
`plug-in for kubectl` is redundant
if there's a way to phrase this differently, it would be awesome like:

> Manage clusters with Giant Swarm

Reply:
Changing to "Handle custom resources with Giant Swarm", like in the overview markdown file